src/firefly/plans/grid_scan.py

```diff
@@ -66,13 +76,19 @@ def setup_ui(self):
         self.stop_line_edit.setPlaceholderText("Stop…")
         self.layout.addWidget(self.stop_line_edit)
 
-        # Fifth item, snake checkbox
+        # Fifth item, number of scan point
+        self.scan_pts_spin_box = QtWidgets.QSpinBox()
+        self.scan_pts_spin_box.setMinimum(1)
+        self.scan_pts_spin_box.setMaximum(99999)
```

Comment:
Why 99999? Do we need a maximum?

Reply:
If this line is removed, the default I get is one digit (so 9)

Reply:
Got it. Makes sense then.